components/ui/Text/Text.tsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import Body from "./Body";
+import Caption from "./Caption";
+import Caption2 from "./Caption2";
+import Footnote from "./Footnote";
+import Headline from "./Headline";
+import Largetitle from "./Largetitle";
+import Overline from "./Overline";
+import Subheadline from "./Subheadline";
+import Subtitle from "./Subtitle";
+import Title from "./Title";
+import Title2 from "./Title2";
+import Title3 from "./Title3";
+
+import classnames from "classnames";
+
+type Props = {
+  variant?:
+    | "overline"
+    | "caption"
+    | "body"
+    | "caption2"
+    | "footnote"
+    | "headline"
+    | "largetitle"
+    | "subheadline"
+    | "subtitle"
+    | "title"
+    | "title2"
+    | "title3";
+  children: any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734246)
```suggestion
  children: unknown;
```